llvm/lib/DebugInfo/LogicalView/Readers/LVCodeViewVisitor.cpp

```diff
@@ -1987,6 +1987,7 @@ Error LVLogicalVisitor::visitKnownRecord(CVType &Record, ClassRecord &Class,
   Scope->setName(Class.getName());
   if (Class.hasUniqueName())
     Scope->setLinkageName(Class.getUniqueName());
+  Scope->setBitSize(Class.getSize() * DWARF_CHAR_BIT);
```

Comment:
What is the difference with `Scope->setBitSize` for `LF_BITFIELD (TPI): BitFieldRecord &BF`?

Reply:
That is right. A DIE can have the `xx_bit_size` or `xx_byte_size`.